airmozilla/manage/templates/manage/event_assignments.html

```diff
@@ -54,6 +54,7 @@
     <tbody>
     {% for event in events %}
       <tr>
+       {% if not event.is_removed() %}
```

Comment:
This is not the right place to do the filtering. You should do that deeper down. Like here maybe: https://github.com/blossomica/airmozilla/blob/bug1164936-Removed-events-should-not-appear-in-the-upcoming-assignments-list/airmozilla/manage/views/events.py#L1348